apps/backoffice-v2/src/domains/workflows/hooks/mutations/useUpdateWorkflowByIdMutation/useUpdateWorkflowByIdMutation.tsx

```diff
@@ -48,8 +48,9 @@ export const useUpdateWorkflowByIdMutation = ({ workflowId }: { workflowId: stri
     onSuccess: (data, { action }) => {
       toast.success(t(`toast:${action}.success`));
     },
-    onError: (error, { action }, context) => {
-      toast.error(t(`toast:${action}.error`, { errorMessage: error.message }));
+    onError: (error: Error, { action }, context) => {
```

Comment:
Aslong as a check like `instanceof Error` or a function like `isErrorWithMessage` is used error is of type `unknown`.